pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +265,37 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) parallelPullSupportExists(serializeImagePulls util.BoolFlag) bool {
```

Comment:
Sorry to keep nit picking, but this name doesn't really match what this method is doing. This method is saying "based on the Docker version and the user's intent (if specified), should we do pulls in parallel or not?". So the name should be something like `shouldPullInParallel`.